symbolic-debuginfo/src/pdb.rs

```diff
@@ -670,6 +671,7 @@ impl<'s> Unit<'s> {
                 line: line_info.line_start.into(),
             });
         }
+        lines.sort_by_key(|line| line.address);
```

Comment:
Ah, perfect, I was thinking about submitting a similar patch (also with the first() / last() change below). Thanks!